cmd/limactl/main.go

```diff
@@ -77,6 +77,7 @@ func newApp() *cobra.Command {
 		newDeleteCommand(),
 		newValidateCommand(),
 		newSudoersCommand(),
+		newDiskUsageCommand(),
```

Comment:
Maybe under a `debug` subcommand

https://github.com/lima-vm/lima/issues/443#issuecomment-985221726

Reply:
I think docker put it under `system`